doc/man1/openssl-smime.pod.in

```diff
@@ -19,33 +19,10 @@ B<openssl> B<smime>
 [B<-crlfeol>]
 [B<-I<cipher>>]
 [B<-in> I<file>]
-[B<-attime> I<timestamp>]
```

Comment:
Hmmm... I find it hard to verify that no option got lost on the way. Any hints how to achieve this?

Reply:
I guess look at the "opt_v_synopsis" variable in one window, and compare it against the "-" lines in another?

Reply:
It occurred to me that I could also compare the generated files. But I didn't find the time to do it yet.